dependency_support/systemlibs/com_grail_bazel_toolchain.toolchain.deps.bzl

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 The XLS Authors
```

Comment:
I don't think this file is needed. XLS currently uses the locally installed compiler toolchain by default. If this changes in the future it is possible to override this choice to the local toolchain with a Bazel command line flag.